pkgs/tools/misc/rogauracore/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchgit, libusb1, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "rogauracore";
+  version = "1.6.0";
+
+  src = fetchgit {
+    url = "https://github.com/wroberts/rogauracore";
+    rev = "a872431a59e47c1ab0b2a523e413723bdcd93a6e";
+    sha256 = "sha256-SeG6B9ksWH4/UjLq5yPncVMTYjqMOxOh2R3N0q29fQ0=";
+  };
+
+  buildInputs = [
+    libusb1
+    autoreconfHook
+    #pkgs.boost
```

Comment:
Why?

Reply:
autoreconfHook is needed to use autoreconf and libusb1 to interact with keyboard. These dependencies are mentioned in the original repo

Reply:
Is `#pkgs.boost` mentioned somehow?

Reply:
ah, you mean comments, sorry